codec/codec.go

```diff
@@ -0,0 +1,46 @@
+package codec
```

Comment:
I just had a quick look at the `transport` package. What is there that needs to know about codec? As far as I can see, it only deals with `[]byte`.